changelog.txt

```diff
@@ -14,7 +14,7 @@ that repo.
 # Future
 
 ## Structures
-- corrected `viewscreen_legendsst` being too short
+- realigned `viewscreen_legendsst`
```

Comment:
Usually our convention is "struct: details"

Reply:
fixed.